kornia/augmentation/auto/operations/base.py

```diff
@@ -0,0 +1,167 @@
+from typing import Callable, Dict, List, Optional, Tuple, TypeVar, Union, cast
+
+import torch
+from torch import nn
+from torch.autograd import Function
+from torch.distributions import Bernoulli, RelaxedBernoulli
+
+from kornia.augmentation.base import _AugmentationBase
+from kornia.augmentation.container.image import ImageSequential
+from kornia.core import Tensor
+
+T = TypeVar('T', bound='OperationBase')
+
+
+class OperationBase(nn.Module):
```

Comment:
```suggestion
class OperationBase(Module):
```
module is also exposed to core